docs/src/content/docs/id/guides/sidebar.mdx

```diff
@@ -87,22 +139,22 @@ Grup dapat berisi baik tautan maupun sub-grup lainnya.
 Tambahkan grup menggunakan objek dengan properti `label` dan `items`. `label` akan digunakan sebagai judul untuk grup.
 Tambahkan tautan atau subgrup ke dalam array `items`.
 
-```js
+```js /^\s*(label:|items:)/
 starlight({
 	sidebar: [
-		// Grup tautan dengan label "Panduan".
+		// Sekelompok tautan yang diberi label "Constellations".
 		{
-			label: 'Panduan',
+			label: 'Constellations',
```

Comment:
This label can be safely translated, would it make sense in this language?

If yes, the example below should also be updated too.

Reply:
I see, thank you for notice this. will update soon.